src/test/kotlin/net/maizegenetics/phgv2/cli/ExportHvcfTest.kt

```diff
@@ -0,0 +1,104 @@
+import org.junit.jupiter.api.extension.ExtendWith
+import java.io.File
+import java.util.stream.Collectors
+import kotlin.test.assertEquals
+
+@ExtendWith(TestExtension::class)
+class ExportHvcfTest {
+
+    companion object {
+
+        private val myLogger = LogManager.getLogger(ExportHvcfTest::class.java)
+
+        val exportHvcfDir = "${TestExtension.tempDir}/export-vcfs/"
+        val outputHvcfDir = "${exportHvcfDir}/output/"
+        val inputHvcfDir = "${exportHvcfDir}/input/"
+        val dbPath = "${exportHvcfDir}/tiledb_export_hvcf"
+        val testHvcfFile = inputHvcfDir + "/" + File(TestExtension.smallseqRefHvcfFile).name
+
+        @BeforeAll
+        @JvmStatic
+        fun setup() {
+
+            File(exportHvcfDir).mkdirs()
+            File(outputHvcfDir).mkdirs()
+            File(inputHvcfDir).mkdirs()
+
+            Initdb().createDataSets(dbPath)
+
+            Files.copy(File(TestExtension.smallseqRefHvcfFile), File(testHvcfFile))
+
+            var builder = ProcessBuilder("conda", "run", "-n", "phgv2-conda", "bgzip", testHvcfFile)
+            var redirectError = "$exportHvcfDir/bgzip_error.log"
+            var redirectOutput = "$exportHvcfDir/bgzip_output.log"
+            builder.redirectOutput(File(redirectOutput))
+            builder.redirectError(File(redirectError))
+
+            println("bgzip Command: " + builder.command().stream().collect(Collectors.joining(" ")));
+            var process = builder.start()
+            var error = process.waitFor()
+            if (error != 0) {
+                myLogger.error("bgzip for $testHvcfFile run via ProcessBuilder returned error code $error")
+                throw IllegalStateException("Error running bgzip for reference: $error")
+            }
+
+            builder = ProcessBuilder("conda", "run", "-n", "phgv2-conda", "tabix", "-p", "vcf", "$testHvcfFile.gz")
+            redirectError = "$exportHvcfDir/tabix_error.log"
+            redirectOutput = "$exportHvcfDir/tabix_output.log"
+            builder.redirectOutput(File(redirectOutput))
+            builder.redirectError(File(redirectError))
+
+            println("tabix Command: " + builder.command().stream().collect(Collectors.joining(" ")));
+            process = builder.start()
+            error = process.waitFor()
+            if (error != 0) {
+                myLogger.error("tabix for $testHvcfFile run via ProcessBuilder returned error code $error")
+                throw IllegalStateException("Error running tabix for reference: $error")
+            }
+
+            // phg load-vcf --vcf-dir /Users/tmc46/phg_v2/ --db-path tiledb/ --temp-dir tiledb/temp/
+
+            val result = LoadVcf().test(
+                "--vcf-dir $inputHvcfDir --db-path $dbPath --temp-dir $exportHvcfDir"
+            )
+
+            println("setup: load vcf output: ${result.output}")
+
+            assertEquals(result.statusCode, 0, "status code not 0: ${result.statusCode}")
+
+        }
+
+    }
+
```

Comment:
I would also make a test to verify the Clikt parameters are working correctly.  You can look at some of the other command test files to see how we can do this easily.

Reply:
Do you mean giving incorrect input?

Reply:
Yes if you look at the other testing classes they have a testCliktParams method which makes sure the code errors out when incorrect parameters are given or required parameters are missed.